elliottlib/cli/get_golang_versions_cli.py

```diff
@@ -61,6 +64,20 @@ def get_golang_versions_cli(runtime, advisory_id, default_advisory_type, nvrs, c
         nvrs = [n.strip() for n in nvrs.split(',')]
         return get_nvrs_golang(nvrs)
 
+    toolset_name = 'go-toolset'
+    build_tag = f'{runtime.group_config.branch}-build'
```

Comment:
That is only the default branch. Maybe look at `brew_tag_product_version_mapping` from `erratatool.yml` (and do some `/s/-candidate$/-build/` instead?

Reply:
Joep made a good point.
Another thing is in rhel 7 the package name is `golang-scl-shim`.